### DISCUSSION TITLE
Can this report actual devices to MQTTT

### DISCUSSION THREAD

**Discussion by TheBartoMan** [arachnetech/homebridge-mqttthing#537] [Q&A]
I already have many Homekit devices installed, such as Nest Thermostats etc.

I was hoping to use this plugin to be able to write out all the information from my current devices to MQTTT. (for use in other apps)

Am I missing something? This seems to just add new virtual devices?

**Comment by arachnetech** [comment]
No, this is for adding devices that use MQTT, not reporting existing devices.

**Reply by TheBartoMan** [reply]
I wish there was a product to do the reverse! I use homeseer for my home systems, and would love to get the info from Homebridge.

**Reply by NorthernMan54** [reply]
> I wish there was a product to do the reverse! I use homeseer for my home systems, and would love to get the info from Homebridge.

There are a couple of API's floating around that can extract Homebridge data for use, and there is node-red and the Homebridge-automation node which also works with homebridge data.
